libs/dal/src/repositories/integration/integration.schema.ts

```diff
@@ -0,0 +1,36 @@
+import * as mongoose from 'mongoose';
+import { Schema, Document } from 'mongoose';
+import { schemaOptions } from '../schema-default.options';
+import { IntegrationEntity } from './integration.entity';
+
+const integrationSchema = new Schema(
+  {
+    _applicationId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Application',
+      index: true,
+    },
+    _organizationId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Organization',
+      index: true,
+    },
+    providerId: Schema.Types.String,
+    channel: Schema.Types.String,
+    credentials: {
+      apiKey: Schema.Types.String,
+      secretKey: Schema.Types.String,
+    },
+    active: Schema.Types.Boolean,
```

Comment:
```suggestion
    active: {
      type: Schema.Types.Boolean,
      default: false
    },
```

Maybe something like this? Just in case so we won't have undefined data in the database